paper-icon-button.html

```diff
@@ -117,6 +117,7 @@
       iron-icon {
         --iron-icon-width: 100%;
         --iron-icon-height: 100%;
+        @apply(--iron-icon);
```

Comment:
To be consistent, I think this should be called `--paper-icon-button-icon`. Please also document it in the "Styling" section of the docs